score/volumes/humanitec.score.yaml

```diff
@@ -6,7 +6,26 @@ spec:
   # i.e. one provisioned via the Platform Orchestrator
   # Resources system.
   volumes:
-    tmp-pod:
+    example-1:
       type: emptyDir
       source:
         sizeLimit: "1024Mi"
+    example-2:
+      type: projected
+      configMap:
+        items: 
+        - key: player_initial_lives
+          path: player_initial_lives.properties
+        - key: ui_properties_file_name
+          path: ui_properties_file_name.properties
+  extraVolumes:
+    - name: example-3
+      projected:
```

Comment:
The README says
> All other volume types are declared in the `extraVolumes` section of the Score extension file.

This `extraVolumes` example is again of type `projected`, so I would expect to find it in the `volumes` section.

So is it possible to use _any_ volume type in the `extraVolumes`, including `projected`?

To set the two sections apart, I suggest we use another example type for `extraVolume`, e.g. an [nfs](https://kubernetes.io/docs/concepts/storage/volumes/#nfs).

Reply:
I did not implement one so far, but borrowing the [example from kubernetes.io](https://kubernetes.io/docs/concepts/storage/volumes/#nfs) should be fine. That would make it

```
  extraVolumes:
    - name: example-3
      nfs:
        server: my-nfs-server.example.com
        path: /my-nfs-volume
        readOnly: true
```

Reply:
@jayonthenet Picking this seasoned PR up again in the course of some housekeeping 🧹 I tested the `extraVolumes` setup with an NFS server and it's working fine. Pushed the change to the `humanitec.score.yaml`, and approved.